wgsl/index.bs

```diff
@@ -3443,6 +3529,62 @@ Issue: Which index is used when it's out of bounds?
            (OpAccessChain)
 </table>
 
+#### Component pointer from vector pointer #### {#component-pointer-from-vector-pointer}
+
+<table class='data'>
+  <caption>Getting a pointer to a component from a pointer to a vector</caption>
+  <thead>
+    <tr><th>Precondition<th>Conclusion<th>Description
+  </thead>
+  <tr algorithm="first vector component pointer selection">
```

Comment:
Is this table needed? it's very verbose and repetitive.
Could we say, alternatively, that x/r -> 0, y/b -> 1, z/g -> 2, w/a -> 3, and then just have a single paragraph explaining how that index works for `vec|N|`?

Reply:
I wrote it this way because it's simpler to read and understand.  The parallel structure makes it easier to scan visually.

Using a shorthand like x/r -> 0 and y/b -> 1 etc leaves open the possibility of mixing x with b, which I'd rather not have to explain in a separate location.